src/syscheckd/create_db.c

```diff
@@ -46,7 +46,131 @@ static const char *FIM_EVENT_MODE[] = {
     "whodata"
 };
 
-// Callback
+typedef struct fim_txn_context_s {
+    event_data_t *evt_data;
+} fim_txn_context_t;
+
+
+static cJSON* fim_dbsync_json_event(const char* path,
+                                    const char* diff,
+                                    const cJSON* fim_entry,
```

Comment:
This parameter is unused in this function

Reply:
the `fim_entry` parameter will old the needed values to fill the commented data in the function. It's needed